### PR TITLE
Migrated all screenshots to separate folders in tutorials

### LINKED ISSUE #11805: [Docs] Tutorials : Organizing Tutorial Screenshots
### Current State
I was working on a tutorial and I noticed that all the screenshots are placed in one folder irrespective of the tutorial. It'll be tedious for someone trying to contribute to the tutorial to find the images and update if needed.
image

### Desired State
- My suggestion is for every tutorial to have its own folder and all the screenshot will live there.
- Another thing we could do here is using descriptive file names, which will help with SEO and collaboration too.

**I can work on this!**

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Pratik-050**
**Notes for Reviewers**

This PR fixes #11805
Organized all screenshots in separate folders.

- [ X ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Pratik-050** [comment]
Hello sir please review this PR. I'm contributing for the first time. If any changes are needed please let me know.
@leecalcote @amitamrutiya

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @Pratik-050. I think what was being suggested is not to organize the images under a common `screenshots` folder, but to place the images adjacent to their respective tutorial.

**Comment by lekaf974** [comment]
> Thanks, @Pratik-050. I think what was being suggested is not to organize the images under a common `screenshots` folder, but to place the images adjacent to their respective tutorial.

Yes I think that like suggested in the issue it will be easier to maintain having screenshots near the tutorial

File: docs/pages/guides/tutorials/wordpress-mysql-persistentvolume.md [hunk 1]
```diff
@@ -62,41 +62,40 @@ These YAML files contain the Service definitions, Persistent Volume Claims, and
 
    - Select `Kubernetes Manifest` from the "Design Type" dropdown menu.
 
-        ![Import modal](./screenshots/wp1.png)
+        ![Import modal](./screenshots/wordpress-mysql-persistentvolume/wp1.png)
```
Comment (sangramrath) [inline docs/pages/guides/tutorials/wordpress-mysql-persistentvolume.md:65]: I feel that these folder names are too long. @leecalcote should we use an ID for each tutorial somewhere and then have folder names with that ID?

**Comment by sangramrath** [comment]
> > Thanks, @Pratik-050. I think what was being suggested is not to organize the images under a common `screenshots` folder, but to place the images adjacent to their respective tutorial.
> 
> Yes I think that like suggested in the issue it will be easier to maintain having screenshots near the tutorial

Wouldn't that kind of do the same thing? All tutorial (MD) files are in a single folder, so instead of these individual screenshot folders being inside the screenshots folder, they would be inside the tutorials folder.

File: docs/pages/guides/tutorials/wordpress-mysql-persistentvolume.md (hunk 1 above)
Comment (Pratik-050) [reply docs/pages/guides/tutorials/wordpress-mysql-persistentvolume.md]: Yes sir that'll be better I guess. I've made some changes as @leecalcote sir told. If further changes are needed let me know. Thank you!

**Comment by l5io** [comment]
🚀 Preview for commit 1b2c5e8a6aba19ea73a784988f71aba872c01c2b at: https://66ea767fb2c8a11a2843eff5--meshery-docs-preview.netlify.app

**Comment by lekaf974** [comment]
You're right having screenshots inside a folder with the same name of the tutorial is working as well

**Comment by vishalvivekm** [comment]
@Pratik-050 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 7:30 PM IST (9:00 AM CT). Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

**Comment by Pratik-050** [comment]
@vishalvivekm sorry but this PR was not discussed in the meet. Please let me know if I need to make any further changes. Thanks, I've liked the meet BTW!

**Comment by Pratik-050** [comment]
Hello everyone can you review it once again to give me a feedback if I need to change anything further. @Alero-Awani @leecalcote @sangramrath

**Comment by sangramrath** [comment]
@Pratik-050 I see a folder with the name of the tutorial inside screenshots folder too. Given the new changes, this should be removed isn't it?

**Comment by l5io** [comment]
🚀 Preview for commit 499e89be2ebdba5437a1cf4b4be0ca6fe1f05f61 at: https://66f38c09a70a27b0e40988fd--meshery-docs-preview.netlify.app

**Comment by Pratik-050** [comment]
> @Pratik-050 I see a folder with the name of the tutorial inside screenshots folder too. Given the new changes, this should be removed isn't it? 

Yes sir sorry to overlook that. I've moved everything from screenshot folder.

**Comment by Pratik-050** [comment]
Hello @sangramrath @Alero-Awani  please review this PR once, else It'll become stale.

**Review by sangramrath — APPROVED** [review]
Looks good to me

**Comment by Pratik-050** [comment]
@leecalcote sir please resolve the requested changes.

**Comment by leecalcote** [comment]
@sangramrath and @Pratik-050, thank you much for sticking with this and seeing it through. 👍

**Review by leecalcote — APPROVED** [review]

**Comment by Pratik-050** [comment]
> @sangramrath and @Pratik-050, thank you much for sticking with this and seeing it through. 👍

Thanks sir!!

**Pull request merged**